### PR TITLE
Eloquent js closing chapter4 : NB the preceding PR is still pending for approval, this one is the latest

### PR DESCRIPTION
**Pull request by DGEs2018**
Could you please review the lastest commits ? Thank you.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Traineeship progress
```
Comment (aiwatko) [inline readme.md]: Do you have the intention to add more to this file? :)

Comment (DGEs2018) [reply readme.md]: No, I'm now working on the challenges and finalising them.

Comment (aiwatko) [reply readme.md]: If you don't have intention to add soemthing in this file, maybe it makes sense to remove this file all together? It doesn't seem to bring much value

Comment (DGEs2018) [reply readme.md]: OK, now I see it's the readme.md file you're speaking of : Yusef created that file to show me that any content inside readme.md is automatically displayed unlike other file types. I'll therefore delete it.

**Pull request merged**